recipes/python-minifier/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "python-minifier" %}
+{% set version = "2.5.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/python_minifier-{{ version }}.tar.gz
+  sha256: 81397b9ec5bf1690e76b358c45af4ba6fa01be2739b478d4c1362b1fcdcf790b
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - pyminify=python_minifier.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<3.11
```

Upstream says `setuptools_scm` as well:
```suggestion
    - python >=3.6,<3.11
    - setuptools_scm
```

---

Roger that